projects/manager/src/app/permissions/permissions.component.html

```diff
@@ -14,10 +14,51 @@
         <td mat-cell *matCellDef="let element">{{ element.permission }}</td>
     </ng-container>
 
+    <ng-container matColumnDef="remove">
+        <th mat-header-cell *matHeaderCellDef>Remove</th>
+        <td mat-cell *matCellDef="let element">
+            <button mat-icon-button (click)="removePermission(element)">
+                <mat-icon>delete</mat-icon>
+            </button>
+        </td>
+    </ng-container>
+
     <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
     <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
 </table>
 
 <mat-paginator [pageSize]="5" [pageSizeOptions]="[5, 10, 20]"></mat-paginator>
```

Comment:
Maybe we should add some config settings to the page size options?

Reply:
I don't think this is necessary. It's not like there are some manager instances with different design that would need different page sizes.